stdlib/task/src/main/ballerina/src/task/listener.bal

```diff
@@ -111,14 +111,14 @@ public type Listener object {
 
     # Pauses the `task:Listener` and the attached services.
     #
-    # + return - Returns `task:ListenerError` if an error is occurred while resuming or nil Otherwise
+    # + return - `task:ListenerError` if an error is occurred while pausing or nil otherwise
```

Comment:
```suggestion
    # + return - `task:ListenerError` if an error is occurred while pausing or else nil otherwise
```
Same as above. Also, please refer the template for the correct format of the return statement description.

Reply:
> ```# + return - `task:ListenerError` if an error is occurred while pausing or else nil otherwise```

Shouldn't this be 

```# + return - `task:ListenerError` if an error is occurred while pausing or else nil```

Check the other places as well.

Reply:
Fixed in all places